accounts/tests.py

```diff
@@ -1,3 +1,33 @@
-# from django.test import TestCase
+import pytest
+from django.contrib.auth.models import User
+from .models import UserAccount
 
-# Create your tests here.
+
+@pytest.fixture
+def test_account(db):
+    username = 'testuser'
+    password = 'lvkmdgv32fdbf1b'
+    email = 'testuser@test.com'
+    new_user = User.objects.create_user(
+        username=username, password=password, email=email
+    )
+    return new_user
+
+
+@pytest.mark.django_db()
+def test_user_create(test_account):
+    assert User.objects.count() > 0
+    assert UserAccount.objects.count() > 0
+    assert UserAccount.objects.count() == User.objects.count()
+
+
+@pytest.mark.django_db()
+def test_user_email(test_account):
+    assert hasattr(test_account, 'email')
+    assert len(test_account.email) > 0
```

Comment:
Both tests checking the test_account object but nothing with the created user on DB,
what about `assert User.objects.get(email=test_account.email)`?
Same for `test_user_find` you can use the `User.objects.get` to find the created user.

The `<modelName>.objects.get` can help with compare/verify db-content

Reply:
+1